sdcm/stress_thread.py

```diff
@@ -105,6 +128,30 @@ def validate_and_collect_hdr_file(self):
             HDRFileMissed(message=f"'{self._remote_log_file}' HDR file couldn't copied from loader {self._node.name}",
                           severity=Severity.ERROR).publish()
 
+    # @raise_event_on_failure
```

Comment:
probably not needed. anyway `self._retrieve` is catching all exceptions...

Reply:
we should remove this comment anyhow